src/main.cpp

```diff
@@ -221,6 +293,60 @@ inline LPVOID FindCharacters(void** commFuncs)
     return lpCharTable;
 }
 
+// Source: https://curl.se/mail/lib-2019-06/0057.html
+mbedtls_x509_crt* extract_cert(PCCERT_CONTEXT certificateContext)
+{
+    // TODO: add delete!
```

Comment:
If you change this to a unique_ptr and move it at the return, you don't have to worry about the delete (just pay attention to where you're letting it move out of scope etc.

Reply:
Done -- it turns out I _did_ have a delete but I changed it to use std::unique_ptr anyway. I need more practice using them.